### ISSUE TITLE
Zero return code when an invalid policy is encountered

### ISSUE THREAD

**Issue by rgee0** [duo-labs/parliament#12]
I've just started exploring this tool and how it might be incorporated into a pipeline test phase.  Currently I am using version 0.2.6

Is the zero return code when an invalid policy is encountered by design?  

Using the example in the readme:
```sh
$ parliament --string '{"Version":"2012-10-17","Statement": {"Effect": "Allow","Action":["s3:GetObject"],"Resource": ["arn:aws:s3:::bucket1"]}}'
INVALID - No resources match for s3:GetObject which requires a resource format of arn:*:s3:::*/* for the resource object* - {'filepath': None}

$ echo $?
0
```

Similar tools in json/yaml linting return non-zero when an error is encountered, which makes trapping undesired states easier than parsing the output.

**Comment by 0xdabbad00** [comment]
I hadn't set any return code. This is a good idea. I'll add this in a future release.  You can see in the code currently no return code is set unless bad arguments are given https://github.com/duo-labs/parliament/blob/master/parliament/cli.py#L113

**Comment by 0xdabbad00** [comment]
Resolved in #16, I'll cut a release shortly.
